$MirOS: ports/www/opera/patches/patch-install_sh,v 1.6 2006/09/21 14:37:31 tg Exp $
$OpenBSD: patch-install_sh,v 1.14 2005/11/22 21:44:22 sturm Exp $
--- install.sh.orig	Mon Oct 15 11:12:12 2007
+++ install.sh	Thu Oct 18 21:14:46 2007
@@ -84,7 +84,7 @@ guess_os()
 	FreeBSD|NetBSD|DragonFly) os=AnyBSD;;
 	SunOS*) os=SunOS;;
     esac
-    case $os in AnyBSD|OpenBSD) str_defaultprefix="/usr/local";; esac
+    case $os in AnyBSD|OpenBSD|MirBSD) str_defaultprefix="/usr/local";; esac
 
     machine=`uname -m`
     case $machine in
@@ -121,7 +121,7 @@ set_prefix()
 
     share_dir="${prefix}/share/opera"
     case "$os" in
-	AnyBSD|OpenBSD)
+	AnyBSD|OpenBSD|MirBSD)
 	    exec_dir="${share_dir}/bin"
 	    plugin_dir="${share_dir}/plugins"
 	    man_dir="$prefix/man"
@@ -428,7 +428,7 @@ set_os_spesific()
 		fi
 	;;
 
-	OpenBSD)
+	OpenBSD|MirBSD)
 	    cpf='-f'
 	    mkdirv=''
 	    chmodv=''
@@ -824,7 +824,7 @@ generate_wrapper()
 	    wrapper_sunjava_machine="sparc"
 	;;
 
-	mips*:Linux)
+	mips*:Linux|i386:MirBSD|x86:MirBSD)
 	;;
 
 	ppc:Linux|ppc64:Linux)
@@ -1063,7 +1063,7 @@ for LIBASPELL_DIR in \\
 done"
 
 case "${os}" in
-	AnyBSD|OpenBSD)
+	AnyBSD|OpenBSD|MirBSD)
 wrapper_contain="${wrapper_contain}
 
 # Make sure the compat libraries are found
@@ -1342,7 +1342,7 @@ run_install()
     then
 	# System wide configuration files
 	config_dir="/etc"
-	if can_write_to "$config_dir"
+	if false can_write_to "$config_dir"
 	then
 	    echo
 	    echo "System wide configuration files:"
@@ -1372,7 +1372,7 @@ run_install()
 	then
 	    echo
 	    echo "Shortcut icons will be ignored if installed with the prefix \"$prefix\"."
-	    con_firm "Do you want to (try to) install them in default locations" || bool_icons=0
+	    bool_icons=0
 	fi
 
 	if test "${bool_icons}" -ne 0
